CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.28.0
+
+* Allow `auto_start_producers` and `allow_topic_auto_creation` to be configurable for brod clients. If configuration of either of these values is desired, update your producer or consumer configs.
```

Comment:
Nice! Do you think it's worth telling what the default is if they're not specified? Maybe it's clear enough for someone to just look at the code 🤔